packages/create/src/recipes/remix/index.ts

```diff
@@ -41,10 +50,15 @@ const recipe: Recipe = {
         "@remix-run/dev": "*",
         "@types/react": "^18.2.20",
         "@types/react-dom": "^18.2.7",
+        "@typescript-eslint/eslint-plugin": "^6.7.4",
+        "@typescript-eslint/parser": "^6.7.4",
+        eslint: "^8.38.0",
+        "eslint-import-resolver-typescript": "^3.6.1",
         "eslint-plugin-import": "^2.28.1",
         "eslint-plugin-jsx-a11y": "^6.7.1",
         "eslint-plugin-react": "^7.33.2",
         "eslint-plugin-react-hooks": "^4.6.0",
+        typescript: "^5.1.6",
```

Comment:
Didn't check the other new dependencies, but is there a reason why we're referring to an older version of `typescript` here?

Reply:
not sure if sth can break in the future if we use "*" in package.json

Reply:
Yeah, that's a good question. Let's just follow their version ranges for now.